### DISCUSSION TITLE
Projects must list all files or use an 'include' pattern.

### DISCUSSION THREAD

**Discussion by GetWrecked99** [nrwl/nx#26648] [Questions]
Hi, 
I have encountered a problem in my nx monorepo workspace using nextjs
I have two applications that using shared components

The problem is that when I import a component in my applications, I get this error : "Projects must list all files or use an 'include' pattern"
however the application is working fine and there is no build error as well. I'm wondering what's the issue here?

![image](https://github.com/nrwl/nx/assets/124883615/a811b7f1-d470-456c-ac01-de32aad12f3c)

this is my tsconfig.base paths that works fine : 
        "paths": {
            "@cms/components": ["components/src/index.ts"],
            "@cms/components/server": ["components/src/server.ts"]
        }

**Comment by danielbraghini** [comment]
I have a problem like that, but sharing NestJs libraries:

![image](https://github.com/user-attachments/assets/856e23f5-4c37-4b7b-8c93-cc499011286a)

**Comment by jdnichollsc** [comment]
@GetWrecked99 @danielbraghini hello folks, did you solve that error?
